library/core/src/macros/mod.rs

```diff
@@ -806,7 +806,7 @@ pub(crate) mod builtin {
     /// This macro functions by taking a formatting string literal containing
     /// `{}` for each additional argument passed. `format_args!` prepares the
     /// additional parameters to ensure the output can be interpreted as a string
-    /// and canonicalizes the arguments into a single type. Any value that implements
+    /// and canonicalize the arguments into a single type. Any value that implements
```

Comment:
I believe this was correct before. "format_args prepares &hellip; and canonicalizes &hellip;"